src/component/common/Search/Search.tsx

```diff
@@ -59,7 +61,7 @@ export const Search = ({
     const placeholder = `${customPlaceholder ?? 'Search'} (${hotkey})`;
 
     return (
-        <div className={styles.container}>
+        <div className={styles.container} style={containerStyles}>
```

Comment:
I do agree, and I'm not entirely happy with how it is now in terms of Styled components / sx / style mix in the project now. But I find using classes is difficult now because (a) we'd like to move away from makeStyles since that is now a third party package that may or may not be maintained in the future, and (b) we need our styles to be connected to the theme, which we can't access through regular styles sheets or css modules.